### PR TITLE
API1000 Support for Deployment Plans

### PR DESCRIPTION
**Pull request by AnushaY1916**
### Description
API1000 Support for Deployment Plans

### Issues Resolved
NA

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for go 1.11 + gofmt checks.
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Storge system
 - Storage volume
 - Storage volume attachment
+- Deployment Plans
```
Comment (jyothis-gireesan) [inline CHANGELOG.md]: Alphabetically we have to give this.

Comment (madhav-bharadwaj) [reply CHANGELOG.md]: Its Singular

File: rest/netutil.go [hunk 2]
```diff
@@ -130,11 +130,15 @@ func (c *Client) RestAPICall(method Method, path string, options interface{}) ([
 			return nil, err
 		}
 		log.Debugf("*** options => %+v", bytes.NewBuffer(OptionsJSON))
+		ioutil.WriteFile("jsons.json", OptionsJSON, 0644)
```
Comment (madhav-bharadwaj) [inline rest/netutil.go]: Are these statements required?

File: rest/netutil.go [hunk 3]
```diff
@@ -130,11 +130,15 @@ func (c *Client) RestAPICall(method Method, path string, options interface{}) ([
...
 		req, err = http.NewRequest(method.String(), reqUrl.String(), bytes.NewBuffer(OptionsJSON))
 	} else {
 		req, err = http.NewRequest(method.String(), reqUrl.String(), nil)
 	}
 
+	file, _ := json.MarshalIndent(options, "", " ")
+	ioutil.WriteFile("2inut.json", file, 0644)
```
Comment (madhav-bharadwaj) [inline rest/netutil.go]: same as above

File: rest/netutil.go (hunk 2 above)
Comment (AnushaY1916) [reply rest/netutil.go]: It was for debugging purpose. Now I have removed.

File: rest/netutil.go (hunk 3 above)
Comment (AnushaY1916) [reply rest/netutil.go]: It was for debugging purpose. Now I have removed.

File: CHANGELOG.md (hunk 1 above)
Comment (AnushaY1916) [reply CHANGELOG.md]: Changes done as mentioned

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,6 +15,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - [#153] (https://github.com/HewlettPackard/oneview-golang/issues/153) change in the enclosure.go file for editing  wrong fields
 
 ### Oneview Features supported
```
Comment (madhav-bharadwaj) [inline CHANGELOG.md]: We can maybe change this to `Features Supported` like in python, as deployment plab is a image streamer feature

**Pull request merged**